src/main/java/lotto/model/LottoNumberGenerator.java

```diff
@@ -0,0 +1,30 @@
+package lotto.model;
+
+import java.util.Comparator;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoNumberGenerator {
+
+    private static List<Integer> lottoNumbers = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
```

Comment:
IntStream을 이용하면 하드 코딩이 줄어들 것 같습니다:)

```suggestion
    private static List<Integer> lottoNumbers = IntStream.range(1, 45)
                .boxed()
                .collect(Collectors.toList());
```
여기다가 1,45도 상수화해서 사용하면 더 깔끔해질것같네요:)